models/articles.models.js

```diff
@@ -73,3 +79,30 @@ exports.updateArticleVotes = (incrementVoteBy, articleId) => {
 			});
 	}
 };
+
+exports.selectCommentsByArticleId = (article_id) => {
+	return db
+		.query(
+			`
+	SELECT 
+	comments.body, 
+	comments.votes, 
+	comments.author,
+	comments.comment_id,
+	comments.created_at
+	FROM comments
+	LEFT JOIN articles ON comments.article_id = articles.article_id 
+	WHERE articles.article_id = $1;
+	`,
+			[article_id]
+		)
+		.then((result) => {
+			if (result.rowCount === 0) {
+				return Promise.reject({
+					status: 404,
+					msg: "Comment not found, ID doesn't exist.",
+				});
```

Comment:
This may not work when you add the test I mentioned so you will need to think of a way to ensure an empty array of comments doesn't get rejected if the article exists.